hibernate-core/src/test/resources/org/hibernate/orm/test/querycache/Item.hbm.xml

```diff
@@ -17,7 +17,7 @@
 			<generator class="native"/>
 		</id>
 		<property name="name" not-null="true"/>
-		<property name="description" not-null="true"/>
```

Comment:
I had to make description nullable in the test entity in ordert to test the null mapping. To my understanding, this should not affect the existing tests.

Reply:
This is fine. Could you please also add a test for a very simple `select null` query (like the one the user reported in the Jira) to assure that works as well?